README.md

```diff
@@ -90,6 +90,22 @@ val books: List<Row> = queryFactory.listQuery {
 }
 ```
 
+### Update
+
+Users can perform bulk update through update query.
+* kotlin-jdsl's update does not require from clause. Type T given as generic handles from automatically.
+* According to the JPA specification, update does not support join, fetch, group by, order by, limit.
+* If you want to use an association mapping as a where condition, you must use associate.
```

Comment:
--korean
`associate` 에 연결 링크가 있으면 좋을 것 같습니다. associate 에 대한 설명이 더 밑에 나와 있는 것 같아서 그렇습니다!

--english
It would be nice to have a link to 'associate'. It's because it seems like the description of the associate is at the bottom!

Reply:
applied

-- korean
적용했습니다.